File: 005/prob5.c
//What is the smallest positive number that is evenly 
//divisible by all of the numbers from 1 to 20?
#include <stdio.h>
#include <stdbool.h>

int main() 
{
	int i = 20;
	bool found = false;

	while (!found)
	{
		bool winner = true;
		int j = 0;

		i++;

		while (winner && j < 20)
		{
			j++;
			winner = i % j == 0;
		}

		if (j == 20) found = true;
	}

	printf("%d\n",i);

	return 0;
}

File: 005/prob5.py
#Project Euler Problem 5
#What is the smallest number divisible by each of the numbers 1 to 20?
found = False
maxOfRange = 10
i = maxOfRange

while not found:
	j = int(maxOfRange / 2)
	valid = True
	
	while valid and j < maxOfRange:
		j += 1
		if i % j != 0: valid = False
	
	if valid: 
		found = True
	else:
		i += 1

print i
